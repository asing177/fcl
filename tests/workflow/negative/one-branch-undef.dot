digraph workflow {
edge [color="#000000"]
graph [bgcolor=transparent]


initial [shape=point; width=0.3; style=filled; color="#0e64ce"; label=""]
commonState [shape=ellipse; fontname="Arial"; fontsize=16; style=filled; color=black; fillcolor=white;]
terminal [shape=point; width=0.3; peripheries=2; style=filled; color="#d11010"; label=""]

1[label=<<FONT POINT-SIZE="16">init</FONT><FONT POINT-SIZE="10" COLOR="blue"> </FONT>>shape=box; fontname="Arial"; style=filled; color=black; fillcolor=gray75;]
2[label=<<FONT POINT-SIZE="16">noop</FONT><FONT POINT-SIZE="10" COLOR="blue"> </FONT>>shape=box; fontname="Arial"; style=filled; color=black; fillcolor=gray75;]
3[label=<<FONT POINT-SIZE="16">use</FONT><FONT POINT-SIZE="10" COLOR="blue"> </FONT>>shape=box; fontname="Arial"; style=filled; color=black; fillcolor=gray75;]

initial -> 1 ; 1 -> commonState
initial -> 2 ; 2 -> commonState
commonState -> 3 ; 3 -> terminal



}
