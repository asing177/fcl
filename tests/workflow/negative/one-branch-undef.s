global int x;

@initial
init()
{
  x = 0;
  transitionTo(@commonState);
}

@initial
noop()
{
  transitionTo(@commonState);
}

@commonState
use()
{
  y = x;
  terminate();
}
