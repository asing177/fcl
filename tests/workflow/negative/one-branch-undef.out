Variable "x" undefined at:
  - line 19:7
Variable "y" undefined at:
  - line 19:3
Stack trace leading up to error:
   noop : {initial} -> {commonState}
   use : {commonState} -> {terminal}
